charts/judge/README.md

```diff
@@ -0,0 +1,56 @@
+# JUDGE Helm Chart
+
+This Helm chart deploys JUDGE on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- You've [spoken to us and received access](https://testifysec.com/products) to JUDGE!
```

Comment:
This needs to be very clearly described at the root of this repo that this chart (even those open source) will not work without proper access.